javascript/frameworks/ui5-webcomponents/test/queries/xss-input-dangerouslySetInnerHTML/src/App.tsx

```diff
@@ -0,0 +1,572 @@
+import { useState, useRef, useEffect, useCallback } from 'react';
+import {Input, TextArea, Search, ShellBarSearch, ComboBox, MultiComboBox, Select, DatePicker, DateRangePicker, DateTimePicker, TimePicker, ColorPicker, ColorPaletteItem, CalendarDate, FileUploader, CheckBox, RadioButton, Switch, Option, OptionCustom, RatingIndicator, Slider, ProgressIndicator, StepInput, DynamicDateRange, RangeSlider, Button, MessageViewButton, SegmentedButton, SplitButton, ToggleButton } from "@ui5/webcomponents-react";
+import '@ui5/webcomponents/dist/Assets.js';
+
+function App() {
+  // Input component usage
+  const [inputValue, setInputValue] = useState<string>("");
+  const inputRef = useRef<typeof Input>(null);
+
+  const handleInputChange = useCallback(() => {
+    setInputValue((msg) => inputRef.current?.value || "");  {/* Potentially Unsafe */}
+  }, [setInputValue]);
```

Comment:
These types of labels are problematic since these add to the target code's AST: (`{ /* ... */ }`) is a comment in an added scope block.
```suggestion
  /* `Input`: Accepts unrestricted string */
  const [inputValue, setInputValue] = useState<string>("");
  const inputRef = useRef<typeof Input>(null);

  const handleInputChange = useCallback(() => {
    setInputValue((msg) => inputRef.current?.value || "");  // UNSAFE: Unrestricted string set as content
  }, [setInputValue]);
```